test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 608400
+//   codeDepositCost: 609200
```

Comment:
Isn't increase in code deposit cost a bad thing? Or could we rather tolerate it for some other benefit?